src/Octokit.Webhooks.AzureFunctions/Octokit.Webhooks.AzureFunctions.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Package">
+    <Product>Octokit.Webhooks.AzureFunctions</Product>
+    <Description>GitHub webhook events toolset for .NET</Description>
+    <PackageTags>Octokit;GitHub;Webhooks</PackageTags>
```

Comment:
Add something functions related to the tags?

Reply:
Added `AzureFunctions`